update.json

```diff
@@ -1,6 +1,6 @@
 {
-  "version": "v1.7.1",
-  "versionCode": 171,
-  "zipUrl": "https://raw.githubusercontent.com/backslashxx/bindhosts/master/module.zip",
+  "version": "v1.7.2",
+  "versionCode": 172,
```

Comment:
not yet
either you revert this or this PR has to wait for 172

Reply:
Can't cause `module.zip` no longer exist and if the update link is not changed to the latest version then it will break the updater.

Reply:
still not yet as this will create headaches issues in this tree if I merge this as-is